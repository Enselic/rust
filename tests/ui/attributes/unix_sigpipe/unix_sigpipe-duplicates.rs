// revisions: with_feature without_feature

#![cfg_attr(with_feature, feature(unix_sigpipe))]

#[unix_sigpipe = "sig_dfl"]
#[unix_sigpipe = "sig_dfl"] //~ error: multiple `unix_sigpipe` attributes
fn main() {}
