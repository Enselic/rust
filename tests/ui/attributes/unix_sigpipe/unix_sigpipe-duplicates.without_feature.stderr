error: multiple `unix_sigpipe` attributes
  --> $DIR/unix_sigpipe-duplicates.rs:6:1
   |
LL | #[unix_sigpipe = "sig_dfl"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: remove this attribute
   |
note: attribute also specified here
  --> $DIR/unix_sigpipe-duplicates.rs:5:1
   |
LL | #[unix_sigpipe = "sig_dfl"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

